migrations/m000000_000000_install.php

```diff
@@ -330,6 +330,9 @@ public function up()
         $this->createIndex('class', models\TagAssign::tableName(), 'class');
         $this->createIndex('item_tag', models\TagAssign::tableName(), ['item_id', 'tag_id']);
 
+	    //CONTENT MODULE
```

Comment:
Yeah, you right.
Btw. I have the idea, to put the migration directly into the modules. It would be possible to install and uninstall modules. Give more flexibility and dynamic.